Nodejs/Product/Nodejs/ProjectTemplates/VuejsApp/VuejsApp.vstemplate

```diff
@@ -0,0 +1,45 @@
+<VSTemplate Version="3.0.0" Type="Project" xmlns="http://schemas.microsoft.com/developer/vstemplate/2005">
+  <TemplateData>
+    <Name Package="{FE8A8C3D-328A-476D-99F9-2A24B75F8C7F}" ID="4009"/>
+    <Description Package="{FE8A8C3D-328A-476D-99F9-2A24B75F8C7F}" ID="4010"/>
+    <Icon Package="{FE8A8C3D-328A-476D-99F9-2A24B75F8C7F}" ID="406"/>
+    <ProjectType>JavaScript</ProjectType>
+    <TemplateID>Microsoft.JavaScript.VuejsApp</TemplateID>
+    <SortOrder>130</SortOrder>
+    <CreateNewFolder>true</CreateNewFolder>
+    <DefaultName>VuejsApp</DefaultName>
+    <ProvideDefaultName>true</ProvideDefaultName>
+    <NumberOfParentCategoriesToRollUp>1</NumberOfParentCategoriesToRollUp>
+    <PromptForSaveOnCreation>true</PromptForSaveOnCreation>
+    <PreviewImage>Preview.png</PreviewImage>
+  </TemplateData>
+  <TemplateContent>
+    <Project File="VuejsApp.njsproj" ReplaceParameters="true">
+      <Folder Name="dist" />
+      <Folder Name="public">
+        <ProjectItem>favicon.ico</ProjectItem>
+        <ProjectItem>index.html</ProjectItem>
+      </Folder>
+      <Folder Name="src">
+        <Folder Name="assets" />
+        <Folder Name="components">
+          <ProjectItem>Home.vue</ProjectItem>
+        </Folder>
+        <ProjectItem>App.vue</ProjectItem>
+        <ProjectItem OpenInEditor="true">main.js</ProjectItem>
+      </Folder>
+      <ProjectItem>.babelrc</ProjectItem>
+      <ProjectItem>.postcssrc.js</ProjectItem>
+      <ProjectItem ReplaceParameters="true">package.json</ProjectItem>
+      <ProjectItem ReplaceParameters="true">README.md</ProjectItem>
```

Comment:
you don't need a `jsconfig.js`?

Reply:
Not needed.

Reply:
Although, it's not a bad idea to include it. Let me know if you have a strong opinion on it and I can add it.